vic.api.v0.yaml

```
@@ -1610,7 +1617,69 @@ paths:
           $ref: "#/components/responses/VendorTagDeletedResponse"
         default:
           $ref: "#/components/responses/ErrorResponse"
-
+  /paymentTerms:
+    get:
+      description: |
+        Get a list of payment terms.
+      summary: Get a list of payment terms
+      operationId: listPaymentTerms
+      tags:
+        - PaymentTerms
+      responses:
+        "200":
+          $ref: "#/components/responses/PaymentTermListResponse"
+        default:
+          $ref: "#/components/responses/ErrorResponse"
+    post:
+      description: |
+        Create a new payment term.
+      summary: Create a new payment term
+      operationId: createPaymentTerm
+      tags:
+        - PaymentTerms
+      requestBody:
+        $ref: "#/components/requestBodies/CreatePaymentTermRequest"
+      responses:
+        "201":
+          $ref: "#/components/responses/PaymentTermResponse"
+        "403":
+          $ref: "#/components/responses/ForbiddenResponse"
+        "422":
+          $ref: "#/components/responses/UnprocessableEntityResponse"
+        default:
+          $ref: "#/components/responses/ErrorResponse"
+    put:
+      description: |
+        Update a payment term.
+      summary: Update a payment term
+      operationId: updatePaymentTerm
+      tags:
+        - PaymentTerms
+      requestBody:
+        $ref: "#/components/requestBodies/UpdatePaymentTermRequest"
+      responses:
+        "200":
+          $ref: "#/components/responses/PaymentTermResponse"
+        "403":
+          $ref: "#/components/responses/ForbiddenResponse"
+        "422":
+          $ref: "#/components/responses/UnprocessableEntityResponse"
+        default:
+          $ref: "#/components/responses/ErrorResponse"
+    delete:
+      description: |
+        Delete a payment term.
+      summary: Delete a payment term
+      operationId: deletePaymentTerm
+      tags:
+        - PaymentTerms
+      parameters:
+        - $ref: '#/components/parameters/PaymentTermId'
+      responses:
+        "201":
```

`201` is for created.

We want to return no content 204

```suggestion
        "204":
```